vc_zoom/indico_vc_zoom/task.py

```diff
@@ -0,0 +1,37 @@
+# This file is part of the Indico plugins.
+# Copyright (C) 2020 - 2022 CERN and ENEA
+#
+# The Indico plugins are free software; you can redistribute
+# them and/or modify them under the terms of the MIT License;
+# see the LICENSE file for more details.
+
+from requests.exceptions import HTTPError
+from sqlalchemy.orm.attributes import flag_modified
+
+from indico.core.celery import celery
+from indico.core.db import db
+
+
+def update_state_log(log_entry, failed):
+    log_entry.data['State'] = 'failed' if failed else 'succeeded'
+    flag_modified(log_entry, 'data')
+
+
+@celery.task(plugin='vc_zoom', max_retries=None)
```

Comment:
Does this allow unlimited retries? That may be a bit risky as a failing request would keep failing forever even if the zoom meeting got deleted in the meantime

Reply:
I have the impression there was a reason, but definitely risky.
I've also updated the retry mechanism.